tests/integration-test/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Our convention is to use the `.bash` extension for bash scripts. After all, the shebang here is `#!/bin/bash`.